### DISCUSSION TITLE
LFO's ?

### DISCUSSION THREAD

**Discussion by poncratias** [shanteacontrols/OpenDeck#165] [Ideas]
Hey there, another idea:

Would it be possible to add LFO functionality to knobs etc?

I had that Idea when talking to other musicians about our synths and how some of them are so limited by only 1 LFO or so. 
And I thought it must be possible to build a simple MIDI LFO Device, that just sends out Midi values that change over time.
Thus giving that our synth much more complexity.

So, essentially assign an encoder or potentiometer to be the LFO for a certain CC value, change the value of that CC over time, depending on a set of parameters (e.g. CC 36  change the value from 0-127 up and down over the time of 10 seconds)
maybe be able to change the parameters of an LFO with a switch or button press

parameters an LFO *can* have:

-CC Value
-Shape (Sine, Square, etc)
-Speed/Rate
-Depth 

Arduino LFO projects:

https://www.flickr.com/photos/51276066@N00/5379297611
https://github.com/robertgallup/arduino-DualLFO
https://github.com/DieterVDW/arduino-lfo-adsr

https://create.arduino.cc/projecthub/scraptured/lo-fi-arduino-nano-synth-modules-fb3a66
https://hacksterio.s3.amazonaws.com/uploads/attachments/1186895/lfo_n8XrIge9fi.JPG

**Comment by paradajz** [comment]
This sounds like a ton of work, so no plans for this.
